README.md

```diff
@@ -30,28 +33,27 @@ A [demonstration notebook](https://github.com/jupyter-widgets/ipywidgets/blob/ma
 ## Jupyter Interactive Widgets as a Framework
 
 Besides the widgets already provided with the library, the framework can be
-extended with custom widget libraries.
+extended with the development of **custom widget libraries**. For detailed
+information, please refer to the [ipywidgets documentation](https://ipywidgets.readthedocs.io/en/latest/).
 
-A template project is available in the form of a cookie cutter [here](https://github.com/jupyter-widgets/widget-ts-cookiecutter).
+### Cookiecutter template for custom widget development
 
-This project is meant to help custom widget authors get started with the
-packaging and the distribution of Jupyter interactive widgets.
-
-It produces a project for a Jupyter interactive widget library following the
-current best practices for using interactive widgets. An implementation for a
-placeholder "Hello World" widget is provided.
+A template project for building custom widgets is available as a
+[**cookiecutter**](https://github.com/jupyter-widgets/widget-ts-cookiecutter).
+This cookiecutter project helps custom widget authors get started with the
+packaging and the distribution of their custom Jupyter interactive widgets.
+The cookiecutter produces a project for a Jupyter interactive widget library
+following the current best practices for using interactive widgets. An
+implementation for a placeholder "Hello World" widget is provided as an example.
 
 Popular widget libraries such as
 [bqplot](https://github.com/bloomberg/bqplot),
 [pythreejs](https://github.com/jovyan/pythreejs) and
 [ipyleaflet](https://github.com/ellisonbg/ipyleaflet)
-
-follow exactly the same template and directory structure. They can serve as
+follow exactly the same template and directory structure. They serve as
```

Comment:
I just realize that bqplot now is in its own github organization (bqplot) while ipyleaflet and pythreejs are in the jupyter-widgets organization.

Reply:
I will update the links. Thanks.